app/controllers/recipes_controller.rb

```diff
@@ -1,10 +1,16 @@
 # frozen_string_literal: true
 
 class RecipesController < ApplicationController
+  before_action :set_recipe, only: %i[show destroy]
+
   def index
     @recipes = Recipe.all
   end
 
+  def show
```

Comment:
Just a comment, you don't have to do it but it seems that this are changes from this [PR](https://github.com/roxuy/learningcamp-recipes/pull/9).

What you should did is to point this PR to that one instead of `main` because I'm guessing you create this branch from the previous one. And after PR #9 is merged do a rebase on this one.